WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -409,6 +409,19 @@ public static float getMediaUploadProgressForPost(PostModel postModel) {
         return 1;
     }
 
+    public static float getUploadProgressForMedia(MediaModel mediaModel) {
+        return MediaUploadHandler.getProgressForMedia(mediaModel);
+    }
+
+    public static List<MediaModel> getPendingMediaForPost(PostModel postModel) {
+        for (UploadingPost uploadingPost : sPostsWithPendingMedia) {
+            if (uploadingPost.postModel.getId() == postModel.getId()) {
+                return uploadingPost.pendingMedia;
+            }
+        }
+        return null;
```

Comment:
What do you think about returning `Collections.emptyList()` here instead? Then we can also annotate the method as returning `@NonNull` (and also drop [this check](https://github.com/wordpress-mobile/WordPress-Android/pull/6450/files#diff-38b6aea9cfff5f3e0e4b7389b44de602R435)).